implementations/rust-boon/Dockerfile

```diff
@@ -7,10 +7,11 @@ RUN update-ca-certificates
 WORKDIR /usr/src/myapp
 COPY . .
 
-ENV RUSTFLAGS='-C linker=x86_64-linux-gnu-gcc'
-RUN cargo build --target x86_64-unknown-linux-musl --release
+#ENV RUSTFLAGS='-C linker=x86_64-linux-gnu-gcc'
+RUN cargo build --release
 
-FROM alpine
-COPY --from=builder /usr/src/myapp/target/x86_64-unknown-linux-musl/release/bowtie-rust-boon /usr/local/bin
+FROM debian:bullseye-slim
```

Comment:
Using a debian base image also looks like it takes the image size from ~20MB up to 80MB, so it'd be slightly nice to see if we can get it working on alpine like the rust-jsonschema image. FWIW it's not (only) macOS that the build fails on, CI is running Linux -- it's ARM64 that fails (on either OS) I think.

Either way, if we do end up having to stick with Debian please remove the dead comment.

Reply:
reverted to alpine.

the original Dockerfile was not building on my laptop. so I updated to Debian. anyway alpine is fine for me.